app/controllers/metadata/MetadataController.scala

```diff
@@ -19,7 +19,7 @@ package controllers.metadata
 import connectors.{DataStoreConnector, EmailThrottlerConnector}
 import domain.SDESInputFormats._
 import domain.{Notification, NotificationsForEori}
-import models.{EmailTemplate, HistoricDocumentRequestSearch, SearchResultStatus}
+import models.{EORI, EmailTemplate, HistoricDocumentRequestSearch, SearchResultStatus}
```

Comment:
You have an unused Import warning here for EORI I think.

Unused import
[warn] import models.{EORI, EmailTemplate, HistoricDocumentRequestSearch, SearchResultStatus}